### ISSUE TITLE
Keep the seminars Zoom accessible even post covid

### ISSUE THREAD

**Issue by sr320** [OARS-SAFS/DEI#33]

**Comment by sr320** [comment]
For clarification, this means to continue to record after we are back "in-person"?

**Comment by JLGnotes** [comment]
Yes. There are some folks who cannot make it to the SAFS seminars easily because of their office/lab location and the time of day.
